vpplink/generated/generate.log

```diff
@@ -1,13 +1,64 @@
-VPP Version                 : 24.02-rc0~8-g5374967e3
+VPP Version                 : 24.02-rc0~186-g956665233
 Binapi-generator version    : v0.10.0-dev
-VPP Base commit             : 7d0cee39a gerrit:34726/3 interface: add buffer stats api
+VPP Base commit             : ac49cf920 docs: Remove references to vlan-strip-offload as it is deprecated
 ------------------ Cherry picked commits --------------------
 capo: Calico Policies plugin
 acl: acl-plugin custom policies
 cnat: [WIP] no k8s maglev from pods
 pbl: Port based balancer
-gerrit:40078/3 vnet: allow format deleted swifidx
-gerrit:40090/3 cnat: undo fib_entry_contribute_forwarding
-gerrit:39507/13 cnat: add flow hash config to cnat translation
+gerrit:40244/1 udp: update rx sw_if_index to ip-local selected one
 gerrit:34726/3 interface: add buffer stats api
+cnat: undo fib_entry_contribute_forwarding
+dpdk: bump rdma-core to 49.0
+ipsec: allow receiving encrypted IP packets with TFC padding
+vppinfra : fix alignment issue
+dev: caps and rx/tx offload support
+dpdk: never override devname if set via config
+dpdk-cryptodev: fix crypto-dispatch node stats
+misc: fix tracedump `show graph` out-of-buffer
+crypto-ipsecmb: bump intel-ipsec-mb version to 1.5
+vnet: allow format deleted swifidx
+dpdk: bump to DPDK 23.11
+tests: tracedump test replace hardcoded value
```

Comment:
You might be missing a `git fetch origin -t` in the VPP repository you're using. 
We don't do this automagicaly iirc, and we arguably should.